test/fixtures/macro-locals.js

```diff
@@ -0,0 +1,20 @@
+import * as t from "ts-interface-checker";
+export function checkLocalInterface(input) {
+  // shows that t is renamed
+  var _t = t.createCheckers({
+    LocalInterface: t.iface([], {
+      "foo": "number"
+    })
+  });
+
+  _t.LocalInterface.check(input);
+
+  return input;
+}
+
+function _t2(t) {
```

Comment:
Hmmm.... This would be a problem if it were `export function t()`, right?

Reply:
Nope! Babel's `scope.rename(identifierL string): void` is pretty smart. I updated this test to show that.